app/src/main/java/com/mapbox/maps/testapp/examples/annotation/PointAnnotationActivity.kt

```diff
@@ -118,7 +141,7 @@ class PointAnnotationActivity : AppCompatActivity() {
         }
         // random add symbols across the globe
         val pointAnnotationOptionsList: MutableList<PointAnnotationOptions> = ArrayList()
-        for (i in 0..20) {
+        for (i in 0..2000) {
```

Comment:
is it intended to leave this large number here? or were you just testing? 😄 
perhaps better having an integration test appying 2000 updates and verifying that __all__ calls where executed (and not were skipped).